### PR TITLE
chore(ruby): Update outdated documentation

### PR DESCRIPTION
**Pull request by kaylareopelle**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

During a documentation audit with @homelessbirds last year, [we identified](https://docs.google.com/spreadsheets/d/1waNtEyvfsYrX74WQaMK7Inry5weUzNkWvzKnwHLLWE0/edit#gid=0) some docs that needed updates. 

Changes to some of the outdated docs are included in this PR. 

Relates to https://github.com/newrelic/newrelic-ruby-agent/issues/2235

### CODE REVIEW

File: src/content/docs/apm/agents/ruby-agent/attributes/enable-disable-attributes-ruby.mdx [hunk 1]
```diff
@@ -1081,17 +1081,17 @@ By default, the Ruby agent reports Uniform Resource Identifiers (URIs) to New Re
 * The `url` attribute of [slow SQL traces](/docs/apm/applications-menu/monitoring/view-slow-query-details)
 * The `uri` attribute of [external request segments](/docs/agents/ruby-agent/features/http-client-tracing-ruby)
 
-New Relic recommends having these URIs reported, as they can contain useful debugging information. If your URIs contain sensitive data that you don't want reported, URI reporting can be disabled. For example, sensitive data could include email addresses or account IDs. To disable URI reporting, add any of the above attribute names to the `attributes.exclude` list.
+New Relic recommends having these URIs reported, as they can contain useful debugging information. If your URIs contain sensitive data that you don't want reported, you can disable URI reporting. Sensitive data could include email addresses or account IDs. To disable URI reporting, add any of the above attribute names to the `attributes.exclude` list.
```
Comment (hannahramadan) [inline src/content/docs/apm/agents/ruby-agent/attributes/enable-disable-attributes-ruby.mdx:1084]:
```suggestion
New Relic recommends having these URIs reported, as they can contain useful debugging information. If your URIs contain sensitive data you don't want to report, such as email addresses or account IDs, you can disable URI reporting. To disable URI reporting, add any of the above attribute names to the `attributes.exclude` list.
```

File: src/content/docs/apm/agents/ruby-agent/attributes/ruby-attribute-examples.mdx [hunk 2]
```diff
@@ -130,27 +130,27 @@ foo, bar, request.parameters.foo, request.parameters.bar
...
+attributes.exclude: food*
+attributes.include: food.fruit.*
+transaction_tracer.attributes.exclude: food.fruit.banana
 ```
 
 **Input keys:**
 
 ```
-lang, lang.functional, lang.oo.ruby, lang.oo.python
+food, food.vegetable, food.fruit.apple, food.fruit.banana
```
Comment (hannahramadan) [inline src/content/docs/apm/agents/ruby-agent/attributes/ruby-attribute-examples.mdx:148]: Love this example improvement 🍎

File: src/content/docs/apm/agents/ruby-agent/attributes/ruby-attribute-examples.mdx [hunk 3]
```diff
@@ -130,27 +130,27 @@ foo, bar, request.parameters.foo, request.parameters.bar
 
 In this example, specific input keys are selected for certain output destinations and excluded from others.
 
-* The `lang.oo.python` key will be excluded only from transaction traces.
-* The `lang` and `lang.functional` keys will be excluded from all destinations.
+* The `food.fruit.banana` key will be excluded only from transaction traces.
```
Comment (hannahramadan) [inline src/content/docs/apm/agents/ruby-agent/attributes/ruby-attribute-examples.mdx:133]: I guess the linter wants us to say something like:
Transaction traces will exclude only the `food.fruit.banana` key.

**Review by jeff-colucci — APPROVED** [review]
Fantastic! Thanks for the updates!

**Pull request merged**